cmd/cli/install.go

```diff
@@ -225,9 +234,114 @@ func (i *installCmd) resolveValues() (map[string]interface{}, error) {
 			return nil, err
 		}
 	}
+
+	for _, val := range parsedArgs {
+		// parses Helm strvals line and merges into a map for the final overrides for values.yaml
+		if err := strvals.ParseInto(val, finalValues); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
@rajdhandus Just checking in to see if you're still working on this.

I think it would simplify things significantly if you iterated over `i.setOptions` here directly vs. parsing them manually first. I know that would mean those values don't get validated but I think that's a worthwhile tradeoff. In the event that a value is defined both with an explicit flag and `--set`, I think the explicit flag should take precedence as that will be validated. We should make that order of precedence clear in the `--help` and that `--set` should only be used when an explicit flag is not available.